editor/icons/source/icon_GUI_tree_arrow_right.svg

```diff
@@ -71,7 +71,7 @@
      id="layer1"
      transform="translate(0,-1040.3622)">
     <path
-       style="fill:none;stroke:#ffffff;stroke-width:2;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:0.39215687"
```

Comment:
these are not supposed to change, you should create another conversion, like #FFFFFF -> #000000